flexget/plugins/input/anilist.py

```diff
@@ -142,14 +144,45 @@ def on_task_input(self, task, config):
                                 and anime.get('format').lower() in selected_formats
                             )
                         ) or 'all' in selected_formats
+
                         if has_selected_type and has_selected_release_status:
+                            try:
+                                ids = task.requests.post(
+                                    'https://relations.yuna.moe/api/ids',
+                                    json={'anilist': anime.get('id')},
+                                ).json()
+                            except RequestException as e:
+                                ids = {}
```

Comment:
Why set `ids` to empty dict here? It's not used ... ?

Also, exception chaining is generally a good idea.

(Besides this PR, holy batman that's a wide exception handing for ValueError)

Reply:
If the request isn't successful and `ids` isn't set then the `ids.get()` calls below will return exceptions.

Reply:
But you're raising PluginWarning immediately after .. which stops the whole plugin execution.